helm/common/deployment-config/templates/_helpers2.tpl

```diff
@@ -0,0 +1,123 @@
+｛｛/*
+- name: shared-tmp 
+  mountPath: /tmp/pod
+*/}}
+{{- define "container.volumeMounts" -}} 
+volumeMounts:
+{{- range $mount := .Values.container.volumeMounts }}
+- name: {{ $mount.name }}-volumne
+  mountPath: {{ $mount. mountPath }}
+  {{- if $mount.subPath }}
+  subPath: {{ $mount.subPath }}
+  {{- end }}
+  {{- if $mount.readOnly }}
+  readOnly: {{ $mount.readOnly }}
+  {{- else }}
+  readOnly: true
+  {{- end }}
+{{- end }}
+{{- end - }}
+
+{{- define "container.command" -}}
+command:
+{{- range $cmd := .Values.container.command }}
+- {{ $cmd }}
+{{- end }}
+{{- end =}}
+
+{{- define "container.args" - }}
+args:
+{{- range $arg := .Values.container.args }}
+- {{ $arg }}
+{{- end }}
+({- end -}}
+
+{{- define "container.securityContext" -}}
+securityContext:
+  runAsUser: {{ default 1001 .Values.container.runAs.user }}
+  runAsGroup: {{ default 1001 .Values.container.runAs.group }}
+  runAsNonRoot: true
+{{- end -}}
+
+{{/*
+Base Deployment
+*/}}
+{{- define "container.env" - }}
+env:
+  {{- toYaml .Values.container.env | nindent 0 - }}
+  {{- if .Values.container.envAddons }}
+   {{- toYaml .Values.container.envAddons | nindent 0 - }}
+  {{- end -}}
+{{- end -}}
+
+{{- define "container.ports" - }}
+{{- if.Values.container.ports -}}
+ports:
+{{- range .Values.container.ports }}
+- containerPort: {{.}} 
+  protocol: TCP
+{{- end }}
+{{- else }}
+ports:
+
+- containerPort: 8080
+  protocol: TCP
+{{- end }}
+{{- end -}}
+
+{{- define "deployment.serviceAccount" - }}
+{{- if .Values.deployment.serviceAccountName }}
+serviceAccountName: {{ Values.deployment.serviceAccountName }}
+{{- else - }}
```

Comment:
remove the _else_ block https://github.com/kishalayb18/DevOps/blob/feature/new/helm/common/deployment-config/templates/_helpers2.tpl#L71-L79